composeApp/src/wasmJsMain/kotlin/Main.kt

```diff
@@ -33,6 +33,7 @@ fun main() {
             null
         }
 
+    // TODO: Understand how to use resources for WAsm
```

Comment:
Could you please elaborate? I thought resources are working the same way on wasm.

Reply:
I just not get in touch, how to get resources when we not in composable context, only using coroutines, like in guide? (https://www.jetbrains.com/help/kotlin-multiplatform-dev/compose-images-resources.html#strings)

Reply:
> We can try do it like this:
> 
> ```kotlin
>     CanvasBasedWindow(title = Res.string.app_name.key, canvasElementId = "ComposeTarget") {
>         RootContent(root)
>     }
> ```

Nope. it gives a key of itself, for this snippet, it returns: "app_name"

Reply:
I think for now we can only use resources in Composable functions. See https://youtrack.jetbrains.com/issue/KT-49981 for a general resources API.

Reply:
Let's write something like:

```
// TODO: Take the title from resources after https://youtrack.jetbrains.com/issue/KT-49981
```